404 on /settings/experiments/assistant:1

When the assistant is not used this results in errors in the console:

`Failed to load resource: the server responded with a status of 404 (Not Found) /settings/experiments/assistant:1 `

version: 0.6.1
clean install
Only the web-thing add-on is installed (removed the Z-Wave and Zigbee add-ons).
I have not touched the smart assistant setting (and thus it's disabled).
Browser: chrome 70


Also:

"assistant experiment setting is not yet set Error: Error getting assistant" in menu.js:51

"Error getting assistant experiment setting Error: Error getting assistant"  in settings.js:711 

the complete console:

```
[webrtc_vad.js status]  Loading webrtc_vad...
stm_web.min.js:1 [webrtc_vad.js status]  Running...
settings/experiments/assistant:1 Failed to load resource: the server responded with a status of 404 (Not Found)
menu.js:51 assistant experiment setting is not yet set Error: Error getting assistant
(anonymous) @ menu.js:51
stm_web.min.js:1 [webrtc_vad.js status]  Webrtc_vad and SpeakToMeVad loaded
stm_web.min.js:1 [webrtc_vad.js status]  
2level.js:214 The specified value "true" is not a valid number. The value must match to the following regular expression: -?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?
set @ level.js:214
thing-model.js:248 Uncaught (in promise) Unavailable property name unit
settings/experiments/assistant:1 Failed to load resource: the server responded with a status of 404 (Not Found)
settings.js:711 Error getting assistant experiment setting Error: Error getting assistant
```

--- a/static/js/api.js
+++ b/static/js/api.js
@@ -337,6 +337,10 @@ const API = {
       })
       .then((response) => {
         if (!response.ok) {
+          if (response.status === 404) {
+            return false;
+          }
+
           throw new Error(`Error getting ${experimentName}`);
         }
 

